op-chain-ops/genesis/setters.go

```diff
@@ -14,6 +14,26 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
+// UntouchablePredeploys are addresses in the predeploy namespace
+// that should not be touched by the migration process.
```

Comment:
I think this will break the check-l2 hh task because it checks for proxies at some of these accounts

Reply:
I'll update the check-l2 script to validate that implementations exist for these rather than proxies - this will make it match the real networks more.

Reply:
This is done.